.github/workflows/test.yml

```diff
@@ -19,26 +19,5 @@ jobs:
           bundler-cache: true
       - name: Install dependencies
         run: bundle install
-      - name: Set ENV for codeclimate (pull_request)
```

Comment:
Wait, it's been removed already https://github.com/chartmogul/chartmogul-ruby/commit/3b972430ce7a875b1e2f337a4e45f8459c6703df. Are you on the latest main?